### PR TITLE
CTCP-1526: Added expires element

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

**Review by jackheslop96 — APPROVED** [review]

**Review by ajchilts — DISMISSED** [review]
LGTM

**Review by ajchilts — APPROVED** [review]

File: app/models/HateoasUserAnswersSummary.scala [hunk 1]
```diff
@@ -18,8 +18,14 @@ package models
 
 import play.api.libs.json.{JsObject, Json}
 
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit.DAYS
+
 object HateoasUserAnswersSummary {
 
+  private def expiresInDays(ttlInDays: Int, createdAt: LocalDateTime): Long =
+    DAYS.between(LocalDateTime.now(), createdAt.plusDays(ttlInDays))
```
Comment (jackheslop96) [inline app/models/HateoasUserAnswersSummary.scala:27]: Do we want to use a `clock` here to ensure they have **exactly** 30 days from the point of creation?

Comment (jackheslop96) [reply app/models/HateoasUserAnswersSummary.scala:27]: Actually I see we're not using a clock at all! That's probably my bad

Comment (jackheslop96) [reply app/models/HateoasUserAnswersSummary.scala:27]: Ah so we currently create the user answers in the frontend and then instantly submit them to the backend. Might be a bit of refactoring to move that logic to the backend. Happy to approve what we've got though 👍

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**